new-pipeline/evo.html

```diff
@@ -4,27 +4,27 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    
+
```

Comment:
Small thing: There are some unrelated whitespace changes in this change, possibly from auto-formatting from your editor?